packages/frontpage/local-infra/docker-compose.yml

```diff
@@ -87,10 +89,23 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  jetstream:
+    container_name: jetstream
+    image: ghcr.io/bluesky-social/jetstream:sha-0ab10bd
```

Comment:
Note: This isn't the most recent version of the jetstream container on ghcr because of reasons discussed in https://github.com/bluesky-social/jetstream/issues/28#issuecomment-2489226867